isis/src/core/tests/unitTests/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# List of classes with old ISIS unitTests
+set(unitTestClasses "FileName"
+                    "IException"
+                    "IString"
+                    "Message"
+                    "Preference"
+                    "Pvl"
+                    "PvlContainer"
+                    "PvlFormat"
+                    "PvlFormatPds"
+                    "PvlGroup"
+                    "PvlKeyword"
+                    "PvlSequence")
+
+foreach (class ${unitTestClasses})
+  set(testFile "${CMAKE_CURRENT_SOURCE_DIR}/${class}/unitTest.cpp")
+  set(truthFile "${CMAKE_CURRENT_SOURCE_DIR}/${class}/${class}.truth")
+  set(executableName "core_unit_test_${class}")
+
+  # Create the test executable
+  add_executable(${executableName} ${testFile})
+  target_link_libraries(${executableName} core)
+
+  # Set up a cmake script which will run the executable
+  # and then check the results against the truth file.
+  set(testScript "${PROJECT_SOURCE_DIR}/cmake/RunUnitTest.cmake")
+  set(fullTestPath $<TARGET_FILE:${executableName}>)
```

Comment:
The location of the test executable changes based on if just core is being built or all of ISIS. This resolves to the proper location regardless of where it ends up.